### PR TITLE
Add a cromwell-http executor

### PR DESCRIPTION
**Pull request by jdidion**

### CODE REVIEW

File: pytest_wdl/executors/cromwell.py [hunk 1]
```diff
@@ -87,7 +90,310 @@ def _read_task_std(path: Path) -> Optional[str]:
...
+    ):
+        super().__init__()
+        self._import_dirs = import_dirs
+
+        if not cromwell_api_url:
+            cromwell_api_url = "http://localhost:8000/api/workflows/v1"
+
+        if cromwell_api_username and cromwell_api_password:
+            self._auth = HTTPBasicAuth(cromwell_api_username, cromwell_api_password)
+            print(self._auth)
```
Comment (michaeltneylon) [inline pytest_wdl/executors/cromwell.py]:
```suggestion
```

**Review by michaeltneylon — DISMISSED** [review]

**Comment by jdidion** [comment]
@michaeltneylon please re-review

File: README.md [hunk 2]
```diff
@@ -48,6 +49,7 @@ The plugins that have extra dependencies are:
 
 * dx: Support for DNAnexus file storage, and for the dxWDL executor.
 * bam: More intelligent comparison of expected and actual BAM file outputs of a workflow than just comparing MD5 checksums.
+* http: Support for executors that use HTTPS protocol to communicate with a remote server (e.g. Cromwell Server)
```
Comment (michaeltneylon) [inline README.md]: This should also be added to the manual.
Looking at that, the manual also now includes the yaml extras but it is not here.

Comment (jdidion) [reply README.md]: should be fixed now

File: pytest_wdl/executors/cromwell_server.py [hunk 3]
```diff
@@ -0,0 +1,234 @@
...
+            wdl_path=wdl_path, import_dirs=self._import_dirs, **kwargs
+        )
+
+        if is_task:
+            raise ValueError(
+                "Cromwell cannot execute tasks independently of a workflow"
+            )
+
+        inputs_dict, _ = read_write_inputs(
+            input_file=kwargs.get("inputs_file"),
```
Comment (michaeltneylon) [inline pytest_wdl/executors/cromwell_server.py]:
```suggestion
            inputs_file=kwargs.get("inputs_file"),
```

**Review by michaeltneylon — APPROVED** [review]

**Pull request merged**